ajax/libs/winstrap/package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "winstrap",
+  "filename": "css/winstrap.min.css",
+  "description": "Winstrap is the official Bootstrap theme for Microsoft design language.",
+  "version": "0.5.11",
+  "authors": [
```

Comment:
Let's follow the author field from upstream.
https://github.com/winjs/winstrap/blob/master/package.json#L9
What do you think?

In package.json, we follow the value from upstream.
In commit message, we cc real person ID.

Thank you.